src/components/state.tsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, ReactNode, useState } from 'react';
+import { Person } from '../types';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const MyContext = createContext<any>(null);
```

Comment:
The context is currently typed as `any`, which is not recommended as it provides no type safety. Consider defining a type for the context that includes all the values you intend to provide, such as `person`, `setPerson`, `error`, `setError`, `isLoading`, `setIsLoading`, `selectedPerson`, and `setSelectedPerson`.